include/libtorrent/utp_socket_manager.hpp

```diff
@@ -104,8 +104,7 @@ namespace libtorrent
 		int loss_multiplier() const { return m_sett.get_int(settings_pack::utp_loss_multiplier); }
 
 		void mtu_for_dest(address const& addr, int& link_mtu, int& utp_mtu);
-		void set_sock_buf(int size);
-		int num_sockets() const { return int(m_utp_sockets.size()); }
+		int num_sockets() const { return m_utp_sockets.size(); }
```

Comment:
was this a merge issue? I would have expected m_utp_sockets.size() to have to be cast to an int